spec/models/in_person_enrollment_spec.rb

```diff
@@ -491,4 +494,36 @@
       expect(failed_enrollment_without_notification.eligible_for_notification?).to eq(false)
     end
   end
+
+  describe 'enhanced_ipp?' do
+    before do
+      allow(IdentityConfig.store).to receive(:usps_eipp_sponsor_id).and_return(usps_eipp_sponsor_id)
+    end
+
+    context 'when the enrollment sponsor ID is equal to the EIPP sponsor ID' do
+      it 'returns true' do
+        user = create(:user)
+        profile = create(:profile, gpo_verification_pending_at: 1.day.ago, user: user)
+        enrollment = create(
+          :in_person_enrollment, user: user, profile: profile,
+                                 sponsor_id: usps_eipp_sponsor_id
+        )
+
+        expect(enrollment.enhanced_ipp?).to be true
+      end
+    end
+
+    context 'when the enrollment sponsor ID does not equal the EIPP sponsor ID' do
+      it 'returns false' do
```

Comment:
Same as above - I think it makes sense to move this setup out of the it block and into the context block.  

You _might_ be able to reduce duplication if you have the user/profile/enrollment setup in the describe block and then define the `sponsor_id` in the two context blocks.

Reply:
- `create` is out of the `it` block - resolved with commit 01eff13e11a461d1485a125175e0e38dbdac6cf1
- I passed a trait (:with_sponsor_id and :enhanced_ipp to add sponsor id) into :in_person_enrolllment to add the sponsor id so I did not need to define it -  resolved with commit b5eb47dfbc230e0157b14ff5b778ac25f281189b